wiki/markdown/run.md

```diff
@@ -0,0 +1,63 @@
+# Run RemRem-Generate Applciation
+
+RemRem-Generate application can be executed with Maven command or with the maven built war file.
+
+## Requirements
+
+- Java
+- Maven
+
+
+## Run RemRem-Generate With Maven Command
+
+1. Change to service directory: 
+`cd service`
+
+1. Execute maven command to build and run RemRem-Generate:
+`mvn spring-boot:run`
+
+
+## Run RemRem-Generate With Java Command
+
+1. Change to service directory: 
+`cd service`
+
+2. Execute maven package command to build the RemRem-Generate war file:
+`mvn package -DskipTests`
+
```

Comment:
Add somethink like
  3. cd ..
otherwise CWD is in service and the commands below don't work.

Reply:
Fixed.